unit-tests/live/frames/test-fps-manual-exposure.py

```diff
@@ -0,0 +1,199 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2025 RealSense, Inc. All Rights Reserved.
+
+# Currently, we exclude D555 as it's failing
+# test:device each(D400*)
+# test:device each(D500*) !D555
+# test:donotrun:!nightly
+
+import pyrealsense2 as rs
+from rspy.stopwatch import Stopwatch
+from rspy import test, log
+import time
+import platform
+
+# This test mirrors test-fps.py but forces manual exposure where supported
+
+global first_frame_seconds
+
+def measure_fps(sensor, profile, seconds_to_count_frames = 10):
```

Comment:
This function is copied from: 
https://github.com/IntelRealSense/librealsense/blob/43e6cc575c65cd3f60627106252ae058647df8b7/unit-tests/live/frames/test-fps.py#L20

And we also have a similar helper function on :
[fps-helper.py](https://github.com/IntelRealSense/librealsense/blob/43e6cc575c65cd3f60627106252ae058647df8b7/unit-tests/live/frames/test-fps.py#L20)

Can you use the helper func here and on the fps test too?